pandas/tests/io/test_parquet.py

```diff
@@ -728,6 +729,13 @@ def test_unsupported_float16(self, pa):
         df = pd.DataFrame(data=data, columns=["fp16"])
         self.check_external_error_on_write(df, pa, pyarrow.ArrowException)
 
+    @pytest.mark.skipif(
```

Comment:
Could you change this to pytest.mark.xfail so we can recognize when this works in the future?

Reply:
thanks, I have changed the decorator to mark.xfail now